### PR TITLE
Tutorial plugin for image parameters

### PR DESCRIPTION
**Pull request by ibutra**
As discussed in https://github.com/matryer/bitbar/pull/261 this is the tutorial for the `templateImage` and `image` parameters

### CODE REVIEW

File: Tutorial/images.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#
+# Include BitBar metadata like this at the top of the file
```
Comment (iosdeveloper) [inline Tutorial/images.sh]: This isn't the scope of this tutorial and should be removed.

File: Tutorial/images.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# (commented out, of course):
+#
```
Comment (iosdeveloper) [inline Tutorial/images.sh]: This isn't the scope of this tutorial and should be removed.

File: Tutorial/images.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+# <bitbar.title>Show templateImages and imagesd</bitbar.title>
```
Comment (iosdeveloper) [inline Tutorial/images.sh]: The last character shouldn't be there.

**Comment by keithamus** [comment]
Shall we merge this when `templateImages` are actually released?

**Comment by iosdeveloper** [comment]
@keithamus [https://github.com/matryer/bitbar/releases/tag/v1.7.0](v1.7.0) was released with image parameters added.

**Comment by iosdeveloper** [comment]
@matryer Can we please get this merged?

**Comment by tresni** [comment]
Looks good to me. Merging.

**Pull request merged**